cpp/src/arrow/flight/flight_benchmark.cc

```diff
@@ -42,7 +42,7 @@ DEFINE_int32(server_port, 31337, "The port to connect to");
 DEFINE_int32(num_servers, 1, "Number of performance servers to run");
 DEFINE_int32(num_streams, 4, "Number of streams for each server");
 DEFINE_int32(num_threads, 4, "Number of concurrent gets");
-DEFINE_int32(records_per_stream, 10000000, "Total records per stream");
+DEFINE_int64(records_per_stream, 10000000, "Total records per stream");
```

Comment:
It's defined as int64 at protobuf
https://github.com/apache/arrow/blob/master/cpp/src/arrow/flight/perf.proto#L26